File: king-admin-java/src/main/java/com/oukingtim/domain/SysRoleMenu.java
package com.oukingtim.domain;

import com.baomidou.mybatisplus.annotations.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * Created by oukingtim
 */
@TableName("sys_role_menu")
@Data
@EqualsAndHashCode(callSuper = false)
public class SysRoleMenu extends BaseModel<SysRoleMenu> {

    private String roleId;
    private String menuId;

    public SysRoleMenu(String menuId) {
        this.menuId = menuId;
    }

    public SysRoleMenu() {
    }
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/domain/SysRoleMenu.kt
package com.oukingtim.domain

import com.baomidou.mybatisplus.annotations.TableName

/**
 * Created by oukingtim
 */
@TableName("sys_role_menu")
class SysRoleMenu() : BaseModel<SysRoleMenu>() {
    var roleId: String? = null
    var menuId: String? = null

    constructor(menuId:String):this(){
        this.menuId = menuId
    }
}